controllers/get.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/get.controllers");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics().then((response) => {
+    res.status(200).send(response);
```

Comment:
This will make a bit of a rod for your back during Front end if you don't send it back on a key of {topics: response}!